packages/models-library/src/models_library/api_schemas_webserver/projects.py

```diff
@@ -118,6 +118,17 @@ class ProjectUpdate(InputSchema):
     quality: dict[str, Any] = FieldNotRequired()
 
 
+class ProjectPatch(InputSchema):
+    name: str = Field(None)
+    description: str = Field(None)
```

Comment:
prune descriptions with a pre-validator. Reuse from those in  https://github.com/itisfoundation/osparc-simcore/blob/f80b0c89e0cb42876cbf625ee2d136eb1b09d0bd/packages/models-library/src/models_library/utils/common_validators.py  or create a newone there

Reply:
I think it is not necessary, I have already used `IDStr` and I allow only string, no None value, I guess that should be sufficient, or?